modules/migration-viewing-migration-plan-log.adoc

```diff
@@ -22,7 +22,7 @@ The command displays the filtered logs of the following pods:
 . In the {mtc-short} web console, click *Migration Plans*.
 . Click the *Migrations* number next to a migration plan to view the *Migrations* page.
 +
-The *Migrations* page displays the migration types associated with the migration plan, for example, *Stage* or *Cutover* for warm migration.
+The *Migrations* page displays the migrations associated with the migration plan, for example, *Stage* or *Cutover*.
```

Comment:
It's nitpicky, but it feels like a run-on sentence.  Obligatory sugg: "...migration plan: *Stage* or *Cutover*."

Reply:
Actually, now I can update this sentence with the actual text (Stage, Cutover, Rollback) and remove "for example).

Reply:
In fact, that whole sentence is pretty much irrelevant, so I am going to remove it.